main/ertp/api/payment.md

```diff
@@ -1,24 +1,70 @@
 # Payment
-Payments hold an amount of digital assets that are in transit.
-Payments can be deposited in purses, split into multiple payments,
-combined, and claimed (getting an exclusive payment and revoking
-access from anyone else). Payments are
-linear, meaning that either a payment has its full original balance,
-or it is used up entirely. It is impossible to partially use a
-payment.
+A `payment` holds digital assets that are in transit or 
+expected to soon be in transit. It can be deposited in `purses`, 
+split into or combined with multiple `payments`, and claimed (getting
+an exclusive `payment` and revoking access from anyone else). 
 
-Payments are often received from other actors and therefore should not be trusted themselves. To get the balance of a payment, use the trusted issuer: issuer.getAmountOf(payment).
+A `payment` is linear, meaning either a `payment` has its full
+original balance, or it is used up entirely. You cannot partially use a
+`payment`. In other words, if a `payment` is 10 Quatloos, you can't
+take out, say, 3 Quatloos from it, leaving the `payment` with 7 Quatloos.
 
-Payments can be converted to Purses by getting a trusted issuer and calling `issuer.makeEmptyPurse()` to create a purse, then `purse.deposit(payment)`.
+However, you can split a `payment` into multiple `payments`, for example 
+into two new `payments` of 3 Quatloos and 7 Quatloos.
+The `split()` operation *burns* (destroys) the original 10 Quatloos `payment`.
+
+`Payments` are often received from other actors and therefore should not be
```

Comment:
the "therefore" isn't right here. It isn't the fact that you got it from someone else that makes them untrusted. It's that they're not self-authenticating. 

Issuers are the other case. If someone gives you an alleged issuer, and says "this is the currency in which I want to be paid" or "this currency represents rights to access my thingamajig", there is no alternate source of authority. If you can get payments or purses containing amounts that match that issuer, then you can participate, otherwise not.